modules/commons/src/main/resources/schemas/neurosciencegraph/commons/entity/v1.0.0.json

```diff
@@ -0,0 +1,95 @@
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/identifier/v0.2.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:EntityShape",
+      "@type": "sh:NodeShape",
+      "property": [
+        {
+          "path": "schema:name",
+          "name": "Name",
+          "description": "The entity name.",
+          "datatype": "xsd:string",
+          "minCount": 1,
+          "maxCount": 1
+        },
+        {
+          "path": "schema:description",
+          "name": "Description",
+          "description": "The entity description",
+          "datatype": "xsd:string",
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:providerId",
+          "name": "ProviderId",
+          "description": "The identifier of the entity as given by a team/laboratory that used it in an experiment. This identifier is local to the team or laboratory.",
+          "datatype": "xsd:string",
+          "maxCount": 1
+        },{
+          "@id":"{{base}}/schemas/neurosciencegraph/commons/identifier/v0.2.0/shapes/WithIdentifierPropertyShape"
+        },
+        {
+          "path": "schema:citation",
+          "name": "Citation",
+          "description": "Citation (s) of publication (s) that describe the entity.",
+          "xone": [
+            {
+              "datatype": "xsd:string"
+            },
+            {
+              "nodeKind": "sh:IRI"
+            }
+          ]
+        },
+        {
+          "path": "schema:license",
+          "name": "License",
+          "description": "License.",
+          "nodeKind": "sh:IRI",
+          "maxCount": 1
+        },
+        {
+          "path": "schema:dateCreated",
+          "name": "Creation date",
+          "description": "Date at which this entity created.",
+          "nodeKind": "xsd:dateTime",
+          "maxCount": 1
+        },
+        {
+          "path": "prov:wasRevisionOf",
+          "name": "Revision of",
+          "description": "The entity for which this entity is a revision of. An entity revision is a specific type of derivation.",
+          "nodeKind": "sh:IRI",
+          "maxCount": 1
+        },{
+          "path": "prov:wasDerivedFrom",
+          "name": "Derived from",
+          "description": "The entity from which this entity was derived from.",
+          "nodeKind": "sh:IRI"
+        },{
+          "path": "prov:wasAttributedTo",
+          "name": "Attributed to",
+          "description": "The agent to which this entity was attributed to.",
+          "nodeKind": "sh:IRI"
+        },
+        {
+          "path": "schema:distribution",
+          "name": "Distribution",
+          "description": "Where the data attached to the entity is published.",
+          "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
With this new version, we could also include a property shape for the path schema:image to allow previews to be attached to any entity?

Reply:
Sure.

Reply:
Done.